dev-client/src/components/modals/ActionsModal.tsx

```diff
@@ -113,10 +115,15 @@ export const ActionsModal = forwardRef<
       backgroundColor="grey.200"
       Footer={Footer}
       {...modalProps}>
-      <Heading variant="h5" textAlign="center">
-        {title}
-      </Heading>
-      <Box height="md" />
+      {hasTitle && (
```

Comment:
question: do you need a hasTItle prop? Would it make sense just to check for title being truthy?

Reply:
It's checked in two places in the jsx and this seemed like it was cleaner.